components/Footer/index.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types'
+import './styles.css'
+
+export const Footer = ({
+	links
+}) => {
+
+	return (
+		<div className='footer-section'>
+			{
+				links.map((item) => (
+					// eslint-disable-next-line react/jsx-key
+					<div className='link-wrap'>
+						<a href='{item.href}'>{item.name}</a>
+					</div>
+				))
+			}
+		</div>
+	)
+}
+
+
+Footer.defaultProps = {
+	links: [
+		{
+			name: 'Home', 
+			href: ''
```

Comment:
@ceciliahangzhou Can we please make the items Dynamic. Also label/value should read from a separate file.